release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -102,6 +102,12 @@ For more information, see xref:../updating/preparing_for_updates/updating-cluste
 
 With this release, you can prevent the `kubeadmin` password from being displayed in the console after the installation by using the `--skip-password-print` flag during cluster creation. The password remains accessible in the `auth` directory.
 
+[id="ocp-4-16-installation-and-update-byoip_{context}"]
+==== Bring your own IPv4 (BYOIP) feature enabled for installation on AWS
+
+With this release, you can enable bring your own IP addresses (BYOIP) feature when installing on AWS by using a public IPv4 pool ID to allocate Elastic IPs (EIPs). You must ensure that you have the xref:../installing//installing_aws/installing-aws-account.adoc#installation-aws-permissions_installing-aws-account[required permissions] to enable BYOIP. For more information, see xref:../installing/installing_aws/installation-config-parameters-aws.adoc#installation-configuration-parameters-optional-aws_installation-config-parameters-aws[Optional AWS configuration parameters].
```

Comment:
```suggestion
With this release, you can enable bring your own public IPv4 addresses (BYOIP) feature when installing on AWS by using a public IPv4 pool ID to allocate Elastic IPs (EIPs). You must ensure that you have the xref:../installing//installing_aws/installing-aws-account.adoc#installation-aws-permissions_installing-aws-account[required permissions] to enable BYOIP. For more information, see xref:../installing/installing_aws/installation-config-parameters-aws.adoc#installation-configuration-parameters-optional-aws_installation-config-parameters-aws[Optional AWS configuration parameters].
```

Reply:
overall LGTM. Should we need to mention the AWS docs here to cross-ref our feature implementation with AWS service that the feature is using?

[1] https://github.com/openshift/openshift-docs/pull/76834/files#diff-14ebe576efb84b2f58b32a0de2996fced237204ae9b091e3cafbd40b65c1ef31R1013
[2] `For more information about bring your own IP addresses (BYOIP) in AWS, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-byoip.html#byoip-onboard[Onboard your BYOIP]`

Reply:
Thanks @mtulio!
Since we are documenting release notes for our docs, which in turn are pointing to the AWS docs, IMO, the current approach seems fine.